api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -420,6 +424,19 @@ message Watchdog {
   type.v3.Percent multikill_threshold = 5;
 }
 
+// Fatal actions to run while crashing. Actions can be safe (meaning they are
+// async-signal safe) or unsafe. We run all safe actions before we run unsafe actions.
+// If using an unsafe action that could get stuck or deadlock, it important to
+// have an out of band system to terminate the process.
+//
+// See the interface for the extensions in include/envoy/server/fatal_action_config.h.
```

Comment:
Prefer providing the name of the interface rather than concrete file path for robustness.

Reply:
Done. Referenced interface as: \``Envoy::Server::Configuration::FatalAction``